python/cuml/common/base.pyx

```diff
@@ -181,8 +180,7 @@ class Base:
         else:
             self.verbose = verbose
 
-        self.output_type = cuml.global_output_type if output_type is None \
-            else _check_output_type_str(output_type)
+        self.output_type = _check_output_type_str(cuml.global_output_type if output_type is None else output_type)
```

Comment:
line too long?

Reply:
Yes... flake should have caught that. Not sure why it passed the style check.

Reply:
Resolved